README.md

```
@@ -75,4 +77,122 @@ yarn hardhat verify --network <network> <address>
+
+The first 5 fields are related to the certificate itself, while the last 3 are used to validate the signature on-chain, as follows:
+
+-  `signer`: address of the account that will sign the certificate. Should have `ADMIN_ROLE` or `OPERATOR_ROLE` in the Lock Contract
+- `authorization`: a unique control number in hex format for each authorization
+- `deadline`: the date until that certificate can be presented to the Lock Contract
+
+## Signing Process
+
+Each signature should also contain the message, the `CertificateAuthorization` types object, and the following domain:
+
+    {
```

Markdownlint (0.37.0)

Expected: fenced; Actual: indented
Code block style

(MD046, code-block-style)